examples/kotlin-multiplatform/build.gradle

```diff
@@ -23,10 +23,12 @@ kotlin {
         compilations.create("builtInExecutor") { associateWith(compilations.main) }
     }
     wasm('wasmJs') { d8() }
-    if (HostManager.host == KonanTarget.MACOS_X64.INSTANCE) macosX64('native')
-    if (HostManager.host == KonanTarget.MACOS_ARM64.INSTANCE) macosArm64('native')
-    if (HostManager.hostIsLinux) linuxX64('native')
-    if (HostManager.hostIsMingw) mingwX64('native')
+    if (HostManager.host == KonanTarget.MACOS_X64.INSTANCE) macosX64('konan')
+    if (HostManager.host == KonanTarget.MACOS_ARM64.INSTANCE) macosArm64('konan')
+    if (HostManager.hostIsLinux) linuxX64('konan')
+    if (HostManager.hostIsMingw) mingwX64('konan')
```

Comment:
Since this is an example, it should be similar to how user multiplatform projects are set up, and they definitely do not use this trick with HostManager anymore.

Reply:
Done. 
By the way, it turns out the plugin does not create benchmark tasks for non-host targets. It just reports that benchmarks for those targets cannot be run in current host, e.g.:
`Skipping benchmarks for 'linuxX64' because they cannot be run on current OS: Expected macos_x64, but was linux_x64`
So the situation is better.